### PR TITLE
Feature/javascript api

### PR DESCRIPTION
**Pull request by deebloo**
Update to 0.9 and use twitters javascript api.

### CODE REVIEW

File: src/twitter-button.html [hunk 1]
```diff
@@ -1,44 +1,64 @@
...
+    </template>
 
-        <template if="{{type == 'mention'}}">
-            <iframe src="https://platform.twitter.com/widgets/tweet_button.html?screen_name={{user}}&amp;type=mention" allowtransparency="true" frameborder="0" scrolling="no"></iframe>
-        </template>
+    <template is="dom-if" if="{{types.hashtag}}">
+      <a href="{{hashtagUrl}}" class="twitter-hashtag-button" data-url$="{{link}}" data-text$="{{text}}"></a>
+    </template>
 
-        <style>
```
Comment (Nevraeka) [inline src/twitter-button.html]: Do we need the iframe height & width?

Comment (deebloo) [inline src/twitter-button.html]: I don't believe so. Especially since the twitter api builds the button for you.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +1,3 @@
 bower_components/
 node_modules/
+.idea
```
Comment (zenorocha) [inline .gitignore]: This should be ignored in your `.gitignore_global` file.

Comment (deebloo) [inline .gitignore]: ah yes you are correct. Will fix

**Pull request merged**